mlflow/store/artifact/cloud_artifact_repo.py

```diff
@@ -236,6 +237,7 @@ def _download_file(self, remote_file_path, local_path):
             not MLFLOW_ENABLE_MULTIPART_DOWNLOAD.get()
             or not file_size
             or file_size < _MULTIPART_DOWNLOAD_MINIMUM_FILE_SIZE
+            or is_fuse_uri(local_path)
```

Comment:
could you add a comment that fuse uri is not compatible with file seek + read ?

Reply:
Will do!